vscode-extension/src/util.ts

```diff
@@ -98,6 +102,15 @@ export async function updateServerState(
   });
 }
 
+export async function updateServerEnv(serverUrl: string, filePath: string) {
+  return await ufetch.post(
+    EDITOR_SERVER_ROUTE_TABLE.SET_ENV_FILE_PATH(serverUrl),
+    {
+      [ENV_FILE_PATH]: filePath,
+    }
+  );
+}
+
```

Comment:
for anyone unaware, 
`[ENV_FILE_PATH]` is syntax equivalent to resolving the value of `ENV_FILE_PATH` as the key for the js object.